_includes/pbc/all/install-features/202505.0/install-the-order-management-promotions-and-discounts-feature.md

```diff
@@ -0,0 +1,157 @@
+| Spryker Core           | {{page.version}} | [Install the Spryker Core feature](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-features/install-the-spryker-core-feature.html)                                       |
+
+### 1) Install the required modules
+
+Install the required modules using Composer:
+
+```bash
+composer require spryker/sales-discount-connector:"^1.1.0" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the following modules have been installed:
+
+| MODULE                                   | EXPECTED DIRECTORY                      |
+|------------------------------------------|-----------------------------------------|
+| SalesDiscountConnector                   | vendor/spryker/sales-discount-connector |
+
+{% endinfo_block %}
+
+### 2) Set up behavior
+
+Set up the following behaviors:
+
+| PLUGIN                               | SPECIFICATION                                                          | PREREQUISITES | NAMESPACE                                                        |
+|--------------------------------------|------------------------------------------------------------------------|---------------|------------------------------------------------------------------|
+| CustomerOrderCountDecisionRulePlugin | Checks if the customer's order count matches the discount's condition. |               | Spryker\Zed\SalesDiscountConnector\Communication\Plugin\Discount |
+
+**src/Pyz/Zed/Discount/DiscountDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Discount;
+
+use Spryker\Zed\Discount\DiscountDependencyProvider as SprykerDiscountDependencyProvider;
+use Spryker\Zed\SalesDiscountConnector\Communication\Plugin\Discount\CustomerOrderCountDecisionRulePlugin;
+
+class DiscountDependencyProvider extends SprykerDiscountDependencyProvider
+{
+    /**
+     * @return list<\Spryker\Zed\DiscountExtension\Dependency\Plugin\DecisionRulePluginInterface>
+     */
+    protected function getDecisionRulePlugins(): array
+    {
+        return array_merge(parent::getDecisionRulePlugins(), [
+            new CustomerOrderCountDecisionRulePlugin(),
+        ]);
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+1. [Create a discount](/docs/pbc/all/discount-management/{{site.version}}/base-shop/manage-in-the-back-office/create-discounts.html) and define its condition as a query string with a `customer-order-count` field.
+2. Log in as a customer with a number of orders that fulfills the discount condition.
+3. Add any product to the cart.
+4. Verify that the discount is applied to the cart.
+
+{% endinfo_block %}
+
+### 3) Enable CustomerOrderCountDecisionRulePlugin discount for merchant orders
+
+Set up the following configuration.
+
+1. Update the required modules using Composer:
+
+```bash
+composer require spryker/merchant-sales-order:"^1.7.0" spryker/merchant-sales-order-extension:"^1.1.0" --update-with-dependencies
```

Comment:
Why is this not part of https://github.com/spryker/spryker-docs/pull/3071/files#diff-aa7f56a9ea5d8d0e149dc0a07a931c7a97f77afdd02386befa4d6f5970bf8133R17?

Reply:
It's optional modules to achieve MP context